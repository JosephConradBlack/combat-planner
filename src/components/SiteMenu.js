import React from 'react';
import { Menu, Container } from 'semantic-ui-react';

const SiteMenu = () => {
  return (
    <Menu inverted secondary pointing size="large">
      <Container>
        <Menu.Item>Combat Planner</Menu.Item>
      </Container>
    </Menu>
  );
};

export default SiteMenu;
