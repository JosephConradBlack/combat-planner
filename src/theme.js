export default {
  h1: {
    fontSize: '6em',
    fontWeight: 300,
    letterSpacing: '0.06em',
    subheader: {
      fontSize: '1em',
      fontWeight: 'lighter'
    }
  }
};
