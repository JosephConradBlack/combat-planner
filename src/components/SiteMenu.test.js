import React from 'react';
import { create } from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';

import theme from '../theme';
import SiteMenu from './SiteMenu';

describe('SiteMenu', () => {
  it('renders', () => {
    const component = create(
      <ThemeProvider theme={theme}>
        <SiteMenu />
      </ThemeProvider>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
