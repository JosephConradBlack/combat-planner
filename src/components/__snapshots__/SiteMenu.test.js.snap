// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteMenu renders 1`] = `
<div
  className="ui large inverted pointing secondary menu"
>
  <div
    className="ui container"
  >
    <div
      className="item"
      onClick={[Function]}
    >
      Combat Planner
    </div>
  </div>
</div>
`;
